esp-rom-sys/README.md

```diff
@@ -0,0 +1,38 @@
+# esp-rom-sys
+
+[![Crates.io](https://img.shields.io/crates/v/esp-rom-sys?labelColor=1C2C2E&color=C96329&logo=Rust&style=flat-square)](https://crates.io/crates/esp-rom-sys)
+[![docs.rs](https://img.shields.io/docsrs/esp-rom-sys?labelColor=1C2C2E&color=C96329&logo=rust&style=flat-square)](https://docs.espressif.com/projects/rust/esp-rom-sys/latest/)
+![MSRV](https://img.shields.io/badge/MSRV-1.84-blue?labelColor=1C2C2E&style=flat-square)
+![Crates.io](https://img.shields.io/crates/l/esp-rom-sys?labelColor=1C2C2E&style=flat-square)
+[![Matrix](https://img.shields.io/matrix/esp-rs:matrix.org?label=join%20matrix&labelColor=1C2C2E&color=BEC5C9&logo=matrix&style=flat-square)](https://matrix.to/#/#esp-rs:matrix.org)
+
+ROM code support.
+
+This includes the definition of ROM code function addresses.
+
+For some targets (ESP32 currently) we will use `libesp_rom.a` from ESP-IDF. The code here corresponds to [ESP-IDF v5.3.1](https://github.com/espressif/esp-idf/blob/v5.3.1/components/esp_rom/patches/esp_rom_spiflash.c)
+
+Since there can be only one version of this crate used in a dependency tree this is expected to never see a major version bump. (i.e. all releases are patch releases)
```

Comment:
Could you update the xtask so that this crate never gets a release beyond 0.1? Only 0.1.x versions.

Reply:
I was thinking about adding the check to esp-rom-sys's build.rs

Reply:
we probably should do both since we could also (for whatever reason) skip the xtask and change it manually